pkg/statistics/handle/globalstats/global_stats_test.go

```diff
@@ -959,3 +959,75 @@ func TestGlobalStatsAndSQLBindingWithConcurrency(t *testing.T) {
 	tk.MustExec("set global tidb_merge_partition_stats_concurrency=2")
 	testGlobalStatsAndSQLBinding(tk)
 }
+
+func TestBlockCheckFMSketch(t *testing.T) {
```

Comment:
It seems that even if I revert your change I can still pass the tests. So maybe the test case doesn't cover the change?

Reply:
You should disable the ```tidb_enable_async_merge_global_stats```.

Reply:
Yes, I have. I've run the TestBlockCheckFMSketch. 
1. I revert your change. 
3. I ran the TestBlockCheckFMSketch, and it still passed.

Reply:
![Uploading image.png…]()

Reply:
I find this problem, only it is reproduced in the real tikv.